pages/404.js

```diff
@@ -0,0 +1,44 @@
+import Link from "next/link";
+import Image from "next/image";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faArrowLeft } from "@fortawesome/free-solid-svg-icons";
+import firstErrorLetter from "../styles/image/404page_4_1.png";
+import middleErrorLetter from "../styles/image/canopas-icon.png";
+import lastErrorLetter from "../styles/image/404page_4_2.png";
+
+export default function Custom404() {
+  return (
+    <>
+      <div className="container flex items-center justify-center flex-col my-0 mx-auto py-60 px-[5%] min-h-[50vh]">
+        <div class="flex flex-row my-auto mx-auto text-center">
+          <Image src={firstErrorLetter} className="h-24" alt="404" />
+          <Image
+            src={middleErrorLetter}
+            className="mx-4 w-[6.5rem]"
+            alt="404"
+          />
+          <Image className="h-24" src={lastErrorLetter} alt="404" />
+        </div>
+        <div className="mt-[2rem] text-center text-[1.4rem] leading-8">
+          The page you’re looking for was moved, renamed or might never existed.
+          {""}
```

Comment:
Is this required?

Reply:
its comes bydefault ,whenever i save the file.

Reply:
It not necessary, you can remove it